CHANGELOG.md

```diff
@@ -10,9 +10,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Valhalla `/expansion` examples in the Jupyter Notebook
+- OpenTripPlanner v2 support for routing & isochrones
```

Comment:
I wonder if it wouldn't be better to specify the supported version for each router. Not in the class name, but in a parameter.